plugins/mirrorStrategy.go

```diff
@@ -375,7 +375,7 @@ func (s *mirrorStrategy) updateLevels(
 		// delete remaining prior offers
 		for i := len(newOrders); i < len(oldOffers); i++ {
 			deleteOp := s.sdex.DeleteOffer(oldOffers[i])
-			deleteOps = append(deleteOps, deleteOp)
+			deleteOps = append(deleteOps, &deleteOp)
```

Comment:
we are appending a reference to the delete-op here, why not use the deleteOp directly?
(when comparing to the usage of `mo` is it because the `mo` variable is already a pointer and we're only ever using pointers for the `txnbuild.Operation`?)

Reply:
Yes that is correct. this is similar to `mo` above